### ISSUE TITLE
poMMo mailing filter

### ISSUE THREAD

**Issue by Renegades** [soonick/poMMo#64]
Hello and thanks for inviting em to gihub.

I need to stress, however, I am quite ignorant of github, never used it before so please bare with my mistakes I may pass on, here at the beginning.

Well, as mentioned, I really think poMMo could kick a lot of b**\* ;-), we just need to polish it up a bit and add functionality. One of it would be a better mailing filter. What I mean by that is ... after you have written up your campaign, (the last step is very poor, still,  and what follows thereafter, too), it would be good if you could filter your entire list(s) and select any part thereof, to whom you want to send your campaign - just similar as you can filter out your subscribers in "manage subscribers".

The groups and sub-groups setup as it is right now we better forget it as in a whole. This is still hanging onto a very old-fashioned and not really effective (not to mention inefficient) way.

= = =

Aforesaid, 
I noticed, when writing up a campaign today and using a template which I used before when writing all manually (before trying a software like poMMo... really the others are all crap anyway, including that dinosaur hack of a failed hack called PHP-List... gosh, anyone using this?) the WYSIWYG editor is quite buggy as well.

First: I needed to edit the config file and add 'Source', otherwise I'd have had no chance to edit everything in source-view ... Which, indeed is the only way to do since when editing in front-end view (HTML), poMMo, somehow, changes the entire source (even strips out part of it) the result is a very stuffed up source and not compellable with newer standards at all. 

Hence, i believe the entire WYSIWYG editing part need a thorough overhaul to level the standards used today in writing up email campaigns.

OH, this is a lot, indeed for an opening. As said, please criticize me and direct me so if, mistakes made already, I can avoid in the future.

Other than that, ... I need also a programmer for some work, so please - if you can suggest or adduce according, I will sure appreciate it!

Cheers

**Comment by soonick** [comment]
Hello Renegades,

First of all I appreciate all your comments and congratulations.

This ticket as is, is very hard to follow so I will have to close it or it will never get looked at. Please open a new ticket with only the description of the feature you are requesting.

Thanks,
Adrian
